packages/webapp/src/routes/credits-purchase/estimated-cost-statutory-biodiversity-credits.js

```diff
@@ -0,0 +1,44 @@
+import creditsPurchaseConstants from '../../utils/credits-purchase-constants.js'
+
+const getLocaleString = num =>
+  num.toLocaleString('en-gb', { style: 'currency', currency: 'GBP', minimumFractionDigits: 0 })
+
+const getRow = ({ tier, unitAmount, cost }) => [
+  { text: tier.toUpperCase() },
+  { text: unitAmount, format: 'numeric' },
+  { text: getLocaleString(cost), format: 'numeric' }
+]
+
+const handlers = {
+  get: async (request, h) => {
+    const creditCosts = request.yar.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_COST_CALCULATION)
+
+    if (!creditCosts) {
+      return h.redirect(creditsPurchaseConstants.routes.CREDITS_PURCHASE_CREDITS_SELECTION)
+    }
+
+    const totalCost = getLocaleString(creditCosts.total)
+
+    return h.view(creditsPurchaseConstants.views.CREDITS_PURCHASE_CREDITS_COST, {
+      totalCost,
+      backLink: creditsPurchaseConstants.routes.CREDITS_PURCHASE_CREDITS_SELECTION,
```

Comment:
I didn't see this backlink in many other routes. Why don't we do this everywhere? Is this why backlink doesn't work in many pages?